@pauliescanlon/gatsby-mdx-embed/src/components/Simplecast/SimplecastEpisode.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FunctionComponent } from 'react'
+import { GeneralObserver } from '../GeneralObserver'
+
+export interface ISimplecastEpisodeProps {
+  /** Simplecast Episode */
+  episodeId: string
+  darkTheme?: boolean
```

Comment:
have a look at `Twitter/Tweet` there's a `theme` prop that can be either `light` or `dark` and defaults to `light` i think i prefer this to the boolean prop `darkTheme`

Reply:
What's passed in is a boolean not a `'light' | 'dark'` option, you want me to take this approach then convert `light`, `dark` to a boolean to pass to the iframe?

```html
<iframe height="200px" width="100%" frameborder="no" scrolling="no" seamless src="https://player.simplecast.com/905d603d-a7e4-4142-9bc3-16012312cc46?dark=false"></iframe>
```

Reply:
Ii'm thinking we try where possible to keep the api consistent between providers. This would keep it inline with `Twitter/Tweet`


```javascript
import React, { FunctionComponent } from 'react'
import { GeneralObserver } from '../GeneralObserver'

export interface ISimplecastEpisodeProps {
  /** Simplecast Episode */
  episodeId: string
  /** Color theme of the Player */
  theme?: 'light' | 'dark'
}

export const SimplecastEpisode: FunctionComponent<ISimplecastEpisodeProps> = ({
  episodeId,
  theme = 'light'
}: ISimplecastEpisodeProps) => {
  return (
    <GeneralObserver>
      <div
        className="simplecast-episode-mdx-embed"
        style={{
          position: 'relative',
          height: '200px',
          width: '100%'
        }}
      >
        <iframe
          title={`simplecast-${episodeId}`}
          src={`https://player.simplecast.com/${episodeId}${
            theme === 'dark' ? '?dark=true' : ''
          }`}
          frameBorder="no"
          scrolling="no"
          seamless
          style={{
            position: 'absolute',
            top: 0,
            left: 0,
            width: '100%',
            height: '100%'
          }}
        />
      </div>
    </GeneralObserver>
  )
}
```

## Stories
```javascript

## Basic usage

<Preview>
  <SimplecastEpisode episodeId="905d603d-a7e4-4142-9bc3-16012312cc46" />
</Preview>

## Theme

<Preview>
  <SimplecastEpisode
    episodeId="905d603d-a7e4-4142-9bc3-16012312cc46"
    theme="dark"
  />
</Preview>
```

## Help

You'll need to update the help stuff too with details about the `theme` prop

Reply:
Updated component API inline with other components and updated help